science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
+
+<figure class="figure-img">
+<img src="./rki_covid_lk_inz_paired_plot-1_en.png" class="figure-img img-fluid" alt="Figure 3: Quotient of 7-day incidences over time (50 pairs of districts)."/>
+<figcaption aria-hidden="true">Figure 3: Quotient of 7-day incidences over time (50 pairs of districts).</figcaption>
+</figure>
+
+We see that the incidences in the districts differed only slightly, only in a few exceptions a factor of 10 is reached. Such a strong influence of the different positivity rates due to local differences in incidence, as described in the example above, can thus be ruled out.
+
+## 4 Positivity rates (local)
+
+We will examine the local positivity rates in the following. Here, we will first consider rapid antigen tests (RAT) and PCR tests together. Since these two groups, as described and explained at the beginning, have very different positivity rates, the total positivity rates also depend on the relative frequency of both types of test. The following figure shows the positivity rate over time, averaged over all districts and nationwide.
+
+<figure class="figure-img">
+<img src="./ppa_test_result_lk_smooth_all-1_en.png" class="figure-img img-fluid" alt="Figure 4: Positivity rate after risk notification over time, smoothed and with confidence interval, averaged over all districts (thick) or nationwide (thin)."/>
+<figcaption aria-hidden="true">Figure 4: Positivity rate after risk notification over time, smoothed and with confidence interval, averaged over all districts (thick) or nationwide (thin).</figcaption>
+</figure>
+
+The positivity rate after a red warning is always greater than after a green warning (low risk with risk exposure). At high incidences, the positivity rate after lower risk without risk exposure is occasionally greater and at the same time more dependent on incidence (rises and falls faster) than after a red warning. A separate consideration of antigen rapid tests and PCR tests can shed light on this.
+
+To give an impression of the local differences, the following figure shows the courses for four randomly selected districts.
+
+
+<figure class="figure-img">
+<img src="./ppa_test_result_lk_smooth_sample-1_en.png" class="figure-img img-fluid" alt="Figure 5: Positivity rate after risk notification over time, smoothed and with confidence interval, for four randomly selected districts."/>
+<figcaption aria-hidden="true">Figure 5: Positivity rate after risk notification over time, smoothed and with confidence interval, for four randomly selected districts.</figcaption>
+</figure>
+
+Here, too, we clearly see the greater dependence of the positivity rate at lower risk without a risk exposure on the incidence: the green curve fluctuates more than the red or yellow curve.
+
+### 4.1 RAT
+
+The time course of the averaging of the positivity rates for rapid antigen tests (RAT) across all districts, broken down by risk notification, is included in the following figure.
```

Comment:
"The time course of the averaging of the positivity rates"
=>
"The time course of averages of the positivity rates"

Reply:
Thanks for your suggestion, it has been implemented in https://github.com/corona-warn-app/cwa-website/pull/3363/commits/4c8e51aead1712b20d50c96a4cd72ac3b0e3bcb5